src/utils/prisma.error.ts

```diff
@@ -0,0 +1,35 @@
+import { Prisma } from '@prisma/client';
+
+const logPrismaError = (e: unknown) => {
+  if (e instanceof Prisma.PrismaClientValidationError) {
```

Comment:
I would just use only if statements, so not else if. It's not wrong at all but it is just cleaner to have if statements makes the code more readable.

Reply:
Changed to "if" blocks